docs/upgrade.html

```diff
@@ -30,6 +30,7 @@ <h4><a id="upgrade_2_1_0" href="#upgrade_2_1_0">Upgrading from 0.8.x, 0.9.x, 0.1
         offset retention period (or the one set by broker) has passed since their last commit.</li>
     <li>The default for console consumer's <code>enable.auto.commit</code> property when no <code>group.id</code> is provided is now set to <code>false</code>.
         This is to avoid polluting the consumer coordinator cache as the auto-generated group is not likely to be used by other consumers.</li>
+    <li>The default value for <code>retries</code> was changed to <code>INT.MAX_VALUE</code>, as we introduces <code>delivery.timeout.ms</code> that sets the upper bound of message deliver timeout. </li>
```

Comment:
Can you mention that this is for the producer? For example:

>  The default value for the producer's <code>retries</code> config was changed to <code>INT.MAX_VALUE</code> ...

Might also be worthwhile including a link to KIP-91

Reply:
updated to include KIP-91 link